resources/redirects

```diff
@@ -45,6 +45,10 @@
 # Any removed old blog page should show the latest blog entries
 ^/[0-9]+/[0-9]+/keycloak-[0-9]+-released=/blog.html
 
+# High-availability
+^/high-availability/single-cluster=/high-availability/single-cluster/introduction
```

Comment:
Having a hard time understanding these redirects. I was expecting the redirects on lines 4 to 9 to be pointing to the new "multi-cluster" pages.

Reply:
If I access `http://localhost:5000/nightly/high-availability/deploy-aws-route53-failover-lambda`, I will be redirected to a 404 page. My confusion is that redirects are global and not per version. 
I guess we have to leave them as they are and fix them after 26.4 is released.

Reply:
Yeah the redirects don't work with `nightly/*` urls it seems.